### ISSUE TITLE
Favourites/recent collaborators list

### ISSUE THREAD

**Issue by VariableClass** [atom/teletype#213]
### Favourites/recent collaborators list 

It would be nice to join or share a portal through some sort of mechanism using GitHub users. 🙂 

For example, selecting the user profile picture of a recently collaborated with/favourite user from within the Teletype interface would send them an e-mail via GitHub and/or flag a notification on the Teletype icon in Atom inviting them to collaborate.

[Atom Discussion Thread](https://discuss.atom.io/t/teletype-favourites-recent-collaborators-list/49752)

**Comment by jasonrudolph** [comment]
Thanks for taking the time to share this idea! :bow:

The concept of a "buddy list" (or something similar) is definitely something that's been on our minds (e.g., #22).

As we prioritize future work, we'll be looking at feature requests like this one to determine where we can deliver the most benefit with the available engineering resources (i.e., full-time Atom engineers, community maintainers, and new contributors). In the meantime, if this is a feature that would make you stand up and dance each time you use Teletype 🕺:smile:, then [adding a :+1: emoji reaction](https://github.com/blog/2119) to the [issue body](https://github.com/atom/teletype/issues/213#issue-274502973) is a great way to let us know.

**Comment by VariableClass** [comment]
Hi @jasonrudolph, I'd say that both this and #214 are covered by #22! I did look through the ___open___ issues to check there was nothing similar already in existence, but not the __closed__ issues. It therefore seemed appropriate that these were two separate feature requests, but I'm happy for these to be closed off if there's some way I can show some ❤️ for #22? 🙂

**Comment by jasonrudolph** [comment]
Hi @VariableClass: Thanks for the follow-up. :bow: I think I might see why you're looking at #213 and #214 as two separate feature requests: Since a portal is permanently closed when the host stops sharing the portal, it seems like #214 might be a prerequisite to #213. First, you need to be able to share/join a portal by specifying a GitHub @username instead of a portal ID (i.e., #214). Once that functionality is in place, we can extend it to let you keep track of the GitHub users that you've recently collaborated with and/or a list of your favorite GitHub users to collaborate with (i.e., #213)?

How does that compare to what you had in mind?

**Comment by VariableClass** [comment]
That sounds more a bit more cohesive, yes 🙂
